tensornetwork/backends/backend_factory.py

```diff
@@ -30,17 +30,78 @@
 #we instantiate each backend only once and store it here
 _INSTANTIATED_BACKENDS = dict()
 
+jax_config_args = dict(
+    dtype=None,  # default: float64 
+    precision=None  # default: jax.lax.Precision.DEFAULT
+)
+numpy_config_args = dict()
+pytorch_config_args = dict()
+tensorflow_config_args = dict()
+symmetric_config_args = dict()
 
+CONFIG_ARGS = {
+    'numpy': numpy_config_args,
+    'jax': jax_config_args,
+    'tensorflow': tensorflow_config_args,
+    'pytorch': pytorch_config_args,
+    'symmetric': symmetric_config_args
+}
+
+
+def configure_backend(backend: Union[Text, abstract_backend.AbstractBackend],
+                      **kwargs):
+  """
+  Globally configure the behaviour of different backends.
+  All backends in _INSTANTIATED_BACKENDS are reconfigured, 
+  and the default config parameters will be changed to `**kwargs`.
+  """
+  if isinstance(backend, abstract_backend.AbstractBackend):
+    backend_name = backend.name
+  else:
+    backend_name = backend
+  for arg, val in kwargs.items():
+    if arg not in CONFIG_ARGS[backend]:
+      raise ValueError(f"unkown config-variable {arg} "
+                       f"for {backend_name} backend")
+    CONFIG_ARGS[backend_name][arg] = val  #TODO(mganahl): add check for val
+
+  if isinstance(backend, abstract_backend.AbstractBackend):
+    backend_name = backend.name
+  else:
+    backend_name = backend
+  # the passed backend does not have to be in _INSTANTIATED_BACKENDS
+  if isinstance(backend, abstract_backend.AbstractBackend):
+    backend.configure(**CONFIG_ARGS[backend_name])
```

Comment:
Depends, in an interactive notebook it could happen. You use the jax backend to do some stuff, then you'd like to try out another configuration. It would be annoying if you have to restart the notebook each time you change the configuration. The other option is to get rid of backend-caching, and just initialize a new backend each time we call `get_backend`. But then `Tensor` and `Node` objects that are already in use will not see the new configuration. This can again be rather confusing in interactive mode.